src/classes/HH_NameSpec_TEST.cls

```diff
@@ -230,7 +226,118 @@ private with sharing class HH_NameSpec_TEST {
         hns.Informal_Greeting_Format__c = 'The {!{!FirstName} {!LastName}} {!FirstName} {!LastName} Family';
         system.assertEquals('The John Doe & Jane Doe John Doe & Sally Smith Sally Smith Family', ns.getExampleName(hns, 'Informal_Greeting_Format__c', listCon));
     }
+
+    /*********************************************************************************************************
+    @description
+    operation:
+        3 contacts, 2 different lastnames in household, contacts' last name is not in the sequence 
+    verify:
+        HH Name, Formal Greeting, and Informal Greeting are correct
+    **********************************************************************************************************/            
+    static testMethod void ThreeContactsTwoLastNamesNotInSequenceWhenTheNameFormatHasLastName() {
+        if (strTestOnly != '*' && strTestOnly != 'ThreeContactsTwoLastNamesNotInSequence') return;
```

Comment:
Shall we remove strTestOnly for all test methods in the class? If so, I will remove the strTestOnly from new test methods. Otherwise, will correct the name.